skyportal/handlers/api/public_pages/public_release.py

```diff
@@ -182,6 +184,10 @@ def patch(self, release_id):
                 groups = session.scalars(
                     Group.select(session.user_or_token).where(Group.id.in_(group_ids))
                 ).all()
+            else:
+                return self.error("Invalid groups")
+            if not groups:
```

Comment:
not sure if this is a git diff issue, but it looks like the `if not groups` is duplicated. 

Also if you have an if -> else, followed by another if, and the else and the second if return the same error, I wonder if this could be reformatted to avoid having the same 2 returns.